analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Operations.cs

```diff
@@ -84,21 +84,53 @@ public void SimpleAssignment_ToLocalVariable_FromTrackedSymbol_VB()
             validator.ValidateTag("B", x => x.HasConstraint(DummyConstraint.Dummy).Should().BeTrue());
         }
 
+        [TestMethod]
+        public void SimpleAssignment_ToLocalField_FromTrackedSymbol_CS()
+        {
+            var validator = SETestContext.CreateCS(@"field = 42; Tag(""Target"", this.field);", new LiteralDummyTestCheck()).Validator;
```

Comment:
The `this.field` in in the other method below
```suggestion
            var validator = SETestContext.CreateCS(@"field = 42; Tag(""Target"", field);", new LiteralDummyTestCheck()).Validator;
```